sources/data-node/api/v2/trading_data.proto

```diff
@@ -271,6 +275,30 @@ message MarketDataConnection {
     PageInfo page_info = 3;
 }
 
+// -- Transfers --
+// Request for transfers
+message GetTransfersRequest {
+    string pubkey = 1;
+    bool is_from = 2;
+    bool is_to = 3;
+}
```

Comment:
Sorry, I didn't notice before, but doesn't this need pagination information if it's going to be paged?